.docker/ci/Dockerfile

```diff
@@ -1,48 +1,85 @@
-# moveit/moveit:melodic-ci
+# moveit/moveit2:crystal-ci
 # Sets up a base image to use for running Continuous Integration on Travis
+FROM ros:crystal-ros-base-bionic
 
-FROM ros:melodic-ros-base
-MAINTAINER Dave Coleman dave@picknik.ai
+LABEL maintainer="mike@picknik.ai"
 
 ENV TERM xterm
 
 # Setup catkin workspace
-ENV CATKIN_WS=/root/ws_moveit
-WORKDIR $CATKIN_WS
+ENV ROS_WS=/root/ws_moveit
+ENV ROS_DISTRO crystal
+
+WORKDIR $ROS_WS
 
 # Commands are combined in single RUN statement with "apt/lists" folder removal to reduce image size
 # https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers
-RUN \
-    mkdir src && \
-    cd src && \
+
+RUN mkdir src && \
+    cd $ROS_WS/src && \
     #
     # Download moveit source, so that we can get necessary dependencies
-    wstool init --shallow . https://raw.githubusercontent.com/ros-planning/moveit/${ROS_DISTRO}-devel/moveit.rosinstall && \
+    wstool init --shallow . https://raw.githubusercontent.com/ros-planning/moveit2/moveit2-ci/moveit.rosinstall && \
     #
     # Update apt package list as previous containers clear the cache
     apt-get -qq update && \
-    apt-get -qq dist-upgrade && \
+    apt-get -qq dist-upgrade -y && \
     #
     # Install some base dependencies
     apt-get -qq install -y \
-        # Some source builds require a package.xml be downloaded via wget from an external location
         wget \
         # Required for rosdep command
         sudo \
-        # Preferred build tools
-        python-catkin-tools \
         clang clang-format-3.9 clang-tidy clang-tools \
-        ccache && \
+        ccache \
+        curl gnupg2 lsb-release \
+        # Some python dependencies for working with ROS2
+        python3-colcon-common-extensions \
+        python3-pip \
+        python-rosdep \
+        python3-wstool \
+        python3-rospkg-modules \
+        python3-rosdistro-modules \
+        && \
+    #
+    # Upgrade Pip
+    python3 -m pip install --upgrade pip && \
```

Comment:
Do we really need to upgrade from the system install pip?
What features are we missing?

Reply:
My understanding is that it's always good to ensure you have the most recent version of pip

Reply:
If we need it for that, Ok, but it's one more thing that will move underneath us, reducing reptabilty. I've had to roll back to system packaged pip a few time, but that just may be me borking my install.

Reply:
![image](https://user-images.githubusercontent.com/3253549/54713647-bd5e1700-4b14-11e9-82d4-76969f028d5d.png)

We really need a requirements.txt :P